### PR TITLE
add homework4

### PR DESCRIPTION
**Pull request by DaniilKorinnyi**
Homework4

### CODE REVIEW

File: homework4/task.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+while True:
```
Comment (klymenok) [inline homework4/task.py:1]: Why do you use an infinite loop here?

**Review by DaniilKorinnyi — COMMENTED** [review]
Correction in the task

File: homework4/task.py (hunk 1 above)
Comment (klymenok) [reply homework4/task.py]: Here you can use an infinite loop, but there is no point of don't that in this case. But still, it was not a mistake, I just wanted to know why did you decide to do this

**Review by klymenok — APPROVED** [review]
Looks good!

**Comment by DaniilKorinnyi** [comment]
> Looks good!

Thanks!

**Pull request merged**